internal/admin/manageUsers.go

```diff
@@ -17,7 +17,7 @@ type UsersList struct {
 }
 
 type messageRequest struct {
-	UserID string `json:"id"`
+	UserID string `json:"tweet_id"`
```

Comment:
this looks wrong, why the field named `UserID` has a json tag `tweet_id`? Also what does it have to do with "add block and unblock for tweets"?